### PR TITLE
#174566179 Prepare app for heroku deployment - [staging]

### PR DESCRIPTION
**Pull request by balldash**
### What does this PR do?

Creates a heroku environment for the app

### Description of Task to be completed?

- create procfile for use on heroku
- create build script to build app on local environment

### How should this be manually tested?

Scroll down to the deployment section of this PR and and click view deployment. It should send you to the review app URL

### What are the relevant pivotal tracker stories?

[#174566179](https://www.pivotaltracker.com/story/show/174566179)

### CODE REVIEW

**Comment by codeliezel** [comment]
@balldash  install cross-env and use it to set environments, also had a start command in your package.json scripts. Also, your procfile should be `web: npm run start`

**Comment by codeliezel** [comment]
@balldash also beg team-mates to review and approve, I want all these done before 11 tomorrow

File: package.json [hunk 1]
```diff
@@ -29,7 +29,13 @@
     "start": "nodemon --exec babel-node ./src/app.js",
```
Comment (codeliezel) [inline package.json:29]: install cross-env and use it to set development test and production environments. This line should be "start:dev"

File: src/tests/index-test.js [hunk 2]
```diff
@@ -22,7 +20,6 @@ describe("app.js should return all endpoints", async () => {
 				});
 		});
 	});
-    
 });
 
-export default describe;
\ No newline at end of file
+export default describe;
```
Comment (codeliezel) [inline src/tests/index-test.js:25]: remove this  export default describe` seems, godspower made a mistake

**Review by codeliezel — CHANGES_REQUESTED** [review]
@balldash go to Heroku dashboard and et automatic deploy to develop branch, and when test passes.. also done forget the PR pipelines and I shouldn't remind you again to write your PT story.

**Comment by codeliezel** [comment]
@balldash delete your build folder and git ignore it

**Comment by codeliezel** [comment]
@balldash resolve ur conflicts

**Comment by balldash** [comment]
> @balldash go to Heroku dashboard and et automatic deploy to develop branch, and when test passes.. also done forget the PR pipelines and I shouldn't remind you again to write your PT story.

@codeliezel I need admin privileges to setup automatic deployment and PR pipelines

**Comment by balldash** [comment]
> @balldash go to Heroku dashboard and et automatic deploy to develop branch, and when test passes.. also done forget the PR pipelines and I shouldn't remind you again to write your PT story.

@codeliezel I need admin access to do this

**Review by Fiyin-Anne — COMMENTED** [review]
How do I download the heroku cli and run heroku local?

**Comment by balldash** [comment]
> How do I download the heroku cli and run heroku local?

@Fiyin-Anne use this link: https://devcenter.heroku.com/articles/heroku-cli

**Comment by codeliezel** [comment]
> How do I download the heroku cli and run heroku local?

@Fiyin-Anne i don't think we need to do that again! @balldash please update your PR

**Comment by balldash** [comment]
> > How do I download the heroku cli and run heroku local?
> 
> @Fiyin-Anne i don't think we need to do that again! @balldash please update your PR description.

@codeliezel I've updated the PR description

**Comment by codeliezel** [comment]
> > > How do I download the heroku cli and run heroku local?
> > 
> > 
> > @Fiyin-Anne i don't think we need to do that again! @balldash please update your PR description.
> 
> @codeliezel I've updated the PR description

Good job

**Review by Bernard-Namangala — COMMENTED** [review]
Works fine just remove the other procfile because there are two procfiles

**Review by Fiyin-Anne — DISMISSED** [review]

**Review by codeliezel — COMMENTED** [review]
- remove the procfile with the small p...
- Its only one procfile you should have!

**Review by md-codes — APPROVED** [review]

**Comment by balldash** [comment]
> * remove the procfile with the small p...
> * Its only one procfile you should have!

@codeliezel I have removed it

**Comment by balldash** [comment]
> Works fine just remove the other procfile because there are two procfiles

@Bernard-Namangala I have removed the duplicate procfile

**Review by donaldcrane — APPROVED** [review]
Great it works!!

**Review by codeliezel — APPROVED** [review]
good job!

**Pull request merged**